Documentation/api-reference.md

```diff
@@ -158,14 +194,16 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
       "eventType": "commerce.orderPlaced",
       "commerce": {
         .....
-      }
+      },
+      "identityMap": customIdentityMap
```

Comment:
can we move these examples under sendEvent syntax, then have one example with Identity map creation + sendEvent